recipes/libsigcpp/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# set and verify flags
+echo $CXX
+CXXFLAGS=
+LDFLAGS=
+CC=
+CFLAGS=
+echo  $CPP
+echo  $CXXCPP
+echo  $M4
+echo  $PERL
+echo  $DOT
+echo  $DOXYGEN
+echo  $XSLTPROC
+echo  $PKG_CONFIG
+PKG_CONFIG_PATH=
+echo  $PKG_CONFIG_LIBDIR
+echo $ACLOCAL_FLAGS
+echo 'int main(){return 0;}'>examples/hello_world.cc
+
+# configure, make, install, check
+./configure  --prefix=$PREFIX CFLAGS='-g -O2' CXXFLAGS='-g -O2' CPP= CXX=
```

Comment:
Though we may change that in the future for 32 and 64 bit support. So, you might want to allow the old `CFLAGS` and `CXXFLAGS` in for that rare case we do set them.